Updates the AWS Beanstalk instructions (#5227)
mattermost/docs: source/install/install-ios-app.rst

1. Open the App Store on your Apple device running iOS 12.1 or later.
2. Search for "Mattermost" and select **GET** to download the app.

> 3. Enter your Apple ID Password or use Touch ID to proceed with the install. You may also be asked to provide an Apple Verification Code received through text/phone call, sign-in notification, or Settings on your trusted device.

Reviewer: ```suggestion
3. Enter your Apple ID Password or use Touch ID to proceed with the install. You may also be asked to provide an Apple Verification Code received through text/phone call, sign-in notification, or **Settings** on your trusted device.
```

> 3. When prompted enter your Apple ID Password or use Touch ID to proceed with the install. You may also be additionally asked to provide an Apple Verification Code received through text/phone call, sign-in notification or from Settings on your trusted device.

Reviewer 2: ```suggestion
3. Enter your Apple ID Password or use Touch ID to proceed with the install. You may also be asked to provide an Apple Verification Code received through text/phone call, sign-in notification, or Settings on your trusted device.
```

4. Open Mattermost from your homescreen and follow the steps below to log in:

>    a. **Enter Server URL:** This is the web address you go to when you want to access Mattermost. It's in the format ``https://domain.com``. You can find the Server URL by asking your System Admin, or by looking at the address bar when your Mattermost browser tab is selected. Click **Connect**.

Reviewer 2: ```suggestion
   a. **Enter Server URL:** This is the web address you go to when you want to access Mattermost. It's in the format ``https://domain.com``. You can find the Server URL by asking your System Admin, or by looking at the address bar when your Mattermost browser tab is selected. Select **Connect** to continue.
```

>    b. In the next screen, provide your **Email or Username** and **Password**. Click **Sign in** to complete the sign in process. This is your account login information provided by your Mattermost Admin.

Reviewer 2: ```suggestion
   b. Enter your **Email or Username** and **Password** provided by your Mattermost System Admin. Select **Sign in**.
```
